src/Nest/QueryDsl/Geo/GeoCoordinateJsonConverter.cs

```diff
@@ -8,7 +8,7 @@ internal class GeoCoordinateJsonConverter : JsonConverter
 		public override bool CanConvert(Type objectType) => true;
 		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
 		{
-			var p = value as GeoCoordinate;
+			var p = (GeoCoordinate)value;
```

Comment:
The null check should still be there because it could be passed a null `GeoCoordinate` still